src/containers/shared/components/Amount.tsx

```diff
@@ -5,23 +5,28 @@ import { localizeNumber } from '../utils'
 import Currency from './Currency'
 
 interface AmountProps {
-  value: {
-    issuer?: string
-    currency: string
-    amount: string
-  }
+  value:
```

Comment:
When it takes a string is it in drops or full XRP?

Reply:
Drops (it basically takes an XRPL Amount now) - I need to fix the conversion